### PR TITLE
[AIRFLOW-6020] fix python 3 KubernetesExecutor iteritems exception

### PR DESCRIPTION
**Pull request by eserdyuk-exos**
Make sure you have checked _all_ steps below.

### Jira

- [x] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-6020
  - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a Jira issue.
  - In case you are proposing a fundamental code change, you need to create an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)).
  - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:  
`kubernetes_executor.py` didn't support Python 3 cause of `dict.iteritems()` using.
It has been fixed by changing `iteritems()` for just `items()`.

### Tests

- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [x] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### CODE REVIEW

File: airflow/executors/kubernetes_executor.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
 from queue import Empty
 from typing import Union
 from uuid import uuid4
+from six import iteritems
```
Comment (mik-laj) [inline airflow/executors/kubernetes_executor.py]: Unfortunately, we do not use the six package in this project. Can you fix it some other way?

Comment (eserdyuk-exos) [reply airflow/executors/kubernetes_executor.py]: It's possible also to use python-future pacakge or to make a Python version check manually.

**Comment by eserdyuk-exos** [comment]
@mik-laj do we really need Python 2 compatibility here? I can just remove `iteritems()` at all and use just `items()`.

**Comment by mik-laj** [comment]
We don't support Python 2 on master brach.

**Comment by eserdyuk-exos** [comment]
Then I'll just change `iteritems()` for `items()` not to compare anything.

**Comment by eserdyuk-exos** [comment]
@mik-laj well, it's fixed now.

**Pull request merged**